dhis-2/dhis-services/dhis-service-dxf2/src/test/java/org/hisp/dhis/dxf2/metadata/objectbundle/ObjectBundleServiceProgramTest.java

```diff
@@ -229,32 +227,6 @@ public void testCreateSimpleProgramReg()
         assertEquals( 2, programTrackedEntityAttributes.size() );
     }
 
-    @Test
```

Comment:
Could we just maintain this test and assert that there is no error?

Reply:
yeah right! I'll do that

Reply:
moreover merging from master restored a deleted file :-\